### PR TITLE
i18n(es): Translated `storyblok.mdx`

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Translated `storyblok.mdx` to spanish

### CODE REVIEW

File: src/content/docs/es/guides/cms/storyblok.mdx [hunk 1]
```diff
@@ -0,0 +1,541 @@
...
+});
+
+const content = data.story.content;
+---
+<StoryblokComponent blok={content} />
+```
+
+Para consultar tus datos, utiliza el gancho `useStoryblokApi`. Esto inicializará una nueva instancia del cliente utilizando la configuración de integración que has establecido.
+
+Para renderizar tu contenido, pasa la propiedad `content` del Story al componente `StoryblokComponent` como una prop `blok`.  Este componente renderizará los Bloks que están definidos dentro de la propiedad `content` property. En este caso, renderizará el componente `BlogPost`.
```
Comment (dreyfus92) [inline src/content/docs/es/guides/cms/storyblok.mdx:219]:
```suggestion
Para renderizar tu contenido, pasa la propiedad `content` del Story al componente `StoryblokComponent` como una prop `blok`. Este componente renderizará los Bloks que están definidos dentro de la propiedad `content` property. En este caso, renderizará el componente `BlogPost`.
```

File: src/content/docs/es/guides/cms/storyblok.mdx [hunk 2]
```diff
@@ -0,0 +1,541 @@
...
+  <head>
+    <title>Storyblok & Astro</title>
+  </head>
+  <body>
+    <StoryblokComponent blok={content} />
+  </body>
+</html>
+```
+
+Para generar páginas para todas tus publicaciones de blog, crea una página  `.astro`  que creará rutas dinámicas. Este enfoque varía dependiendo de si estás utilizando la **generación de sitios estáticos** (la opción predeterminada) o **el renderizado en el lado del servidor**.
```
Comment (dreyfus92) [inline src/content/docs/es/guides/cms/storyblok.mdx:399]:
```suggestion
Para generar páginas para todas tus publicaciones de blog, crea una página `.astro` que creará rutas dinámicas. Este enfoque varía dependiendo de si estás utilizando la **generación de sitios estáticos** (la opción predeterminada) o **el renderizado en el lado del servidor**.
```

File: src/content/docs/es/guides/cms/storyblok.mdx [hunk 3]
```diff
@@ -0,0 +1,541 @@
...
+</html>
+```
+
+Este archivo generará una página para cada historia, con el slug y contenido obtenidos de la API de Storyblok.
+
+#### Renderizado en el lado del servidor
+
+Si has [optado por el modo de SSR](/es/guides/server-side-rendering/#habilitando-ssr-en-su-proyecto), utilizarás rutas dinámicas para obtener los datos de la página desde Storyblok.
+
+Crea un nuevo directorio  `src/pages/blog/` y agrega un nuevo archivo llamado `[...slug].astro` con el siguiente código:
```
Comment (dreyfus92) [inline src/content/docs/es/guides/cms/storyblok.mdx:454]:
```suggestion
Crea un nuevo directorio `src/pages/blog/` y agrega un nuevo archivo llamado `[...slug].astro` con el siguiente código:
```

File: src/content/docs/es/guides/cms/storyblok.mdx [hunk 4]
```diff
@@ -0,0 +1,541 @@
...
+
+##### Vercel
+
+Para configurar un webhook en Vercel:
+
+1. Ve al panel de control de tu proyecto y haz clic en **Settings**. 
+
+2. Bajo la pestaña **Git**, encuentra la sección **Deploy Hooks**. 
+
+3. Proporciona un nombre para tu webhook y la rama en la que deseas activar la compilación. Haz clic en **Add**  y copia la URL generada.
```
Comment (dreyfus92) [inline src/content/docs/es/guides/cms/storyblok.mdx:525]:
```suggestion
3. Proporciona un nombre para tu webhook y la rama en la que deseas activar la compilación. Haz clic en **Add** y copia la URL generada.
```

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
Buen trabajo @Waxer59, te dejo unas sugerencias 😁

**Comment by Waxer59** [comment]
Gracias por la revisión 🚀

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Nice work @Waxer59 & @dreyfus92 💜

**Pull request merged**